scrapy/crawler.py

```diff
@@ -123,17 +161,43 @@ def _setup_crawler_logging(self, crawler):
             crawler.signals.connect(log_observer.stop, signals.engine_stopped)
 
     def stop(self):
+        """
+        Stops simultaneously all the crawling jobs taking place.
+
+        Returns a deferred that is fired when they all have ended.
+        """
         return defer.DeferredList([c.stop() for c in list(self.crawlers)])
 
     @defer.inlineCallbacks
     def join(self):
-        """Wait for all managed crawlers to complete"""
+        """
+        join()
```

Comment:
Yes, but in that case I'd have to add it both to CrawlerRunner and CrawlerProcess (this isn't inherited), and I would have to exclude `join()` from `:members:` and `:inherited-members:` because it would get duplicated otherwise. Looks strange but I think I prefer keeping the signature where it is right now.